File: templates/Uwp/Pages/ListDetails.Prism/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.cs
//{**
// This code block adds the method `GetListDetailsDataAsync()` to the SampleDataService of your project.
//**}
namespace Param_RootNamespace.Core.Services
{
    public class SampleDataService : ISampleDataService
    {
//^^
//{[{

        // Remove this once your ListDetails pages are displaying real data.
        public async Task<IEnumerable<SampleOrder>> GetListDetailsDataAsync()
        {
            await Task.CompletedTask;
            return AllOrders();
        }
//}]}
    }
}

File: code/TemplateStudioForUWP/Templates/_comp/_shr/List._VB/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.vb
'{**
' This code block adds the method `GetListDetailsDataAsync()` to the SampleDataService of your project.
'**}
Namespace Services
    Public Module SampleDataService
'^^
'{[{

        ' Remove this once your ListDetails pages are displaying real data.
        Public Async Function GetListDetailsDataAsync() As Task(Of IEnumerable(Of SampleOrder))
            Await Task.CompletedTask

            Return AllOrders()
        End Function
'}]}
    End Module
End Namespace
